tests/workflows/helloWorld.wdl

```diff
@@ -40,17 +44,16 @@ task HelloWorldTask {
 
     # ------------------------------------------------
     # Input args:
-
+    input {
     # Required:
-
-     # Runtime Options:
-     String docker
-     Int? mem
-     Int? preemptible_attempts
-     Int? disk_space_gb
-     Int? cpu
-     Int? boot_disk_size_gb
-
+        String docker
+    # Runtime Options:
+        Int mem
```

Comment:
miniwdl failed this workflow because it didn't like operating on optional int with an int
`mem * 1024`